examples/verify-quorum/host/src/main.rs

```diff
@@ -155,3 +154,12 @@ async fn main() -> eyre::Result<()> {
     println!("successfully generated and verified proof for the program!");
     Ok(())
 }
+
+// We can't use `public_key_to_address()` from `reth_primitives`` because Reth depend on
+// `secp256k1` 0.30 while we are still on 0.29.
+pub fn public_key_to_address(public: PublicKey) -> Address {
+    // strip out the first byte because that should be the SECP256K1_TAG_PUBKEY_UNCOMPRESSED
+    // tag returned by libsecp's uncompressed pubkey serialization
```

Comment:
```suggestion
    // Strip out the first byte because that should be the SECP256K1_TAG_PUBKEY_UNCOMPRESSED
    // tag returned by libsecp's uncompressed pubkey serialization.
```